lms/envs/common.py

```diff
@@ -3465,3 +3465,4 @@ def _make_locale_paths(settings):
 # Once a user has watched this percentage of a video, mark it as complete:
 # (0.0 = 0%, 1.0 = 100%)
 COMPLETION_VIDEO_COMPLETE_PERCENTAGE = 0.95
+COMPLETION_BY_VIEWING_DELAY_MS = 5000
```

Comment:
I don't like that the default is `5000` in `common.py` and `0` in the python code. Can we make it consistent, or define it in only one place?